14_Rotate Letter Encription.py

```diff
@@ -5,8 +5,21 @@
 # choose to as long as rotate returns the correct string.
 # Note that n can be positive, negative or zero.
 
-def rotate(string, number):    
-    # Your code here
+def rotate(string, number):
+    newstring=''
+    for letter in string:
+        if 'a'<=letter<='z':

```

Comment:
Ahaan, very good. Would you find another way to accomplish the same using mod operator?

Reply:
you want me use mod operator for if condition here right ?